frontend/src/components/SuperUser/SuperUserAdmin/Charts/GraphLineChart.jsx

```diff
@@ -0,0 +1,104 @@
+import {
+  CartesianGrid,
+  LineChart,
+  XAxis,
+  Line,
+  YAxis,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from "recharts";
+import { useAuth } from "../../../../contexts/AuthContext";
+import { useEffect, useState } from "react";
+function GraphLineChart({ fetchTransitions }) {
+  const [fetchData, setFetchData] = useState([]);
+  const { token } = useAuth();
+  const apiCall = async () => {
+    try {
+      const url = "/api/branches/getallbranch";
+      const response = await fetch(url, { headers: { Authorization: token } });
+      if (!response.ok) {
+        throw new Error();
+      }
+      const responseJson = await response.json();
+      setFetchData(responseJson);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+  useEffect(() => {
+    apiCall();
+  }, []);
+  const getBranchName = (value) => {
+    if (fetchData.length > 0) {
```

Comment:
when call branch got array that is why i don't need to find Object.entrie just normal or higher iteration